core/services/chainlink/config_test.go

```diff
@@ -857,6 +859,8 @@ Keeper = 1005
 [EVM.GasEstimator.BlockHistory]
 BatchSize = 17
 BlockHistorySize = 12
+CheckInclusionBlocks = 18
+CheckInclusionPercentile = 19
```

Comment:
Sidebar for follow up: we could consider a special type for percentile fields, if explicit is more readable :thinking: 
```suggestion
CheckInclusionPercentile = '19%'
```